### ISSUE TITLE
iCubGenova11 S/N:050 – Right thumb proximal tendon broken

### ISSUE THREAD

**Issue by fedeceola** [robotology/icub-tech-support#1702]
### Robot Name 🤖

iCubGenova11 S/N:050

### Request/Failure description

As per title, while running some experiments the proximal tendon of the right thumb broke.

### Detailed context

As per title, while running some experiments the proximal tendon of the right thumb broke. I cannot run my experiments and I have an upcoming deadline for a paper submission.

### How does it affect you?

I cannot run my experiments and I have an upcoming deadline for a paper submission.

**Comment by fedeceola** [comment]
Hi, are there any plans to fix this? I'd need to do some tests by the end of the week.

**Comment by sgiraz** [comment]
Hi @fedeceola,

Today @GiacomoVassallo13 started the maintenance activity on `iCubGenova11`, further updates will follow tomorrow morning

cc @AntonioConsilvio

**Comment by fedeceola** [comment]
Hi, with @AntonioAzocar we just tried to run the robot. It cannot complete the calibration of the right thumb. The motor of the proximal joint is moving but the finger doesn't close. Furthermore, the tactile sensor of the thumb is not working.

cc @sgiraz @GiacomoVassallo13 @AntonioConsilvio

**Comment by fedeceola** [comment]
Hi, actually I couldn't test the joint because, due to the issue in #1723, the right hand couldn't finish the calibration.

**Comment by AntonioConsilvio** [comment]
Hi @fedeceola! Today I checked the status of the robot.

All tendons were intact, but the right thumb proximal joint didn't work because the connector was quite detached from the MC4-PLUS. 

Now all joints are working properly:


https://github.com/robotology/icub-tech-support/assets/114915464/2b54c7b3-6c5f-4d0c-9bfc-72cb78d86fa1



In addiction I fixed the configuration files, see the PR:

- https://github.com/robotology/robots-configuration/pull/626

cc @sgiraz
